docs/dashboard.asciidoc

```diff
@@ -15,86 +15,90 @@ image:images/Dashboard_example.png[Example dashboard]
 [[dashboard-getting-started]]
 == Building a Dashboard
 
+If you haven't yet indexed data into {es} or created an index pattern, 
+you'll be prompted to do so as you follow the steps for creating a dashboard.  
+Or, you can use the prebuilt flight data set, available from the 
```

Comment:
Would it be better to just say a generic message like "use one of the prebuilt sample data sets". That way, if the flight data is ever removed, this comment still makes sense?